pkg/apis/openebs.io/v1alpha1/cstorpool_cluster.go

```diff
@@ -66,6 +67,14 @@ type CStorPoolClusterSpec struct {
 	// Pools is the spec for pools for various nodes
 	// where it should be created.
 	Pools []PoolSpec `json:"pools"`
+	// DefaultResources are the compute resources required by the cstor-pool
+	// container.
+	// If the resources at PoolConfig is not specified, this is written
+	// to CSPI PoolConfig.
+	DefaultResources *v1.ResourceRequirements `json:"resources"`
+	// AuxResources are the compute resources required by the cstor-pool pod
+	// side car containers.
+	AuxResources v1.ResourceRequirements `json:"auxResources"`
```

Comment:
Can we have `AuxResources` as pointers like `DefaultResources` ?

Reply:
This is left intentional. Any motivation behind making this pointer too?

Reply:
i think the idea is nil check in case of `DefaultResources` (so we used pointer), why not same for auxResources as well ?